ChemoTracker/package.json

```diff
@@ -34,24 +35,24 @@
     "react-native": "^0.48",
     "react-native-collapsible": "^0.10.0",
     "react-native-datepicker": "^1.6.0",
+    "react-native-dimension": "^1.0.6",
     "react-native-elements": "^0.11.2",
     "react-native-fetch-blob": "^0.10.8",
     "react-native-fm-picker": "0.0.5",
+    "react-native-form-generator": "^0.9.9",
     "react-native-image-picker": "^0.25.6",
     "react-native-linear-gradient": "^2.4.0",
-    "react-native-dimension": "^1.0.6",
     "react-native-material-bottom-navigation": "^0.5.2",
     "react-native-modal-filter-picker": "^1.3.2",
     "react-native-modal-picker": "0.0.16",
     "react-native-picker-dropdown": "^0.0.1",
     "react-native-progressive-input": "^1.0.3",
     "react-native-router-flux": "^4.0.0-beta.21",
+    "react-native-tab-view": "^0.0.74",
     "react-native-tableview-simple": "^0.17.1",
     "react-native-tag-input": "0.0.18",
     "react-native-vector-icons": "^4.5.0",
     "react-navigation": "^1.0.0-beta.11",
-    "react-select": "^1.2.1",
-    "react-native-tab-view": "^0.0.74",
-    "apsl-react-native-button": "^3.0.0"
```

Comment:
Search uses this! Don't delete.
(make sure all the other features still work when you delete things off here)